llvm/include/llvm/SYCLLowerIR/ModuleSplitter.h

```diff
@@ -140,7 +144,14 @@ class ModuleDesc {
   Properties Props;
 
   ModuleDesc(std::unique_ptr<Module> &&M, StringRef Name = "TOP-LEVEL")
-      : M(std::move(M)), IsTopLevel(true), Name(Name) {}
+      : M(std::move(M)), IsTopLevel(true), Name(Name) {
+    // DeviceLib module doesn't include any entry point,it can be constructed
+    // using ctor without any entry point related parameter.
+    for (auto Fn : SYCLDeviceLibs) {
+      if (Name == std::string(Fn))
+        IsSYCLDeviceLib = true;
```

Comment:
We should add `break;`, there is no need to scan other function names once we've found what we were looking for

Reply:
Done.
Thanks very much.